### DISCUSSION TITLE
Pennsylvania and Rhode Island

### DISCUSSION THREAD

**Discussion by kuriwaki** [kuriwaki/cvr_harvard-mit_scripts#172] [General]
@mreece13, I'm noticing that Alleghany, PA and Rhode Island seemed at one point in the data but not in metadata/constests.csv. Harvard has collected RI (probably 0-1% discrepancy), not Alleghany. Was wondering if you plan to release it or if there is a reason it cannot.

**Comment by mreece13** [comment]
Yes, PA, RI, and WV were generated using Kevin's python script separately. I've gone ahead and ported those over to the MEDSL pipeline and will build them with the next run of the pipeline. Good catch, I had forgotten about these.

**Reply by kuriwaki** [reply]
Looks like these are in now! thanks.
